### PR TITLE
Hard code special cases to allow in translation key linter

### PR DESCRIPTION
**Pull request by pkong-ds**
ref DEV-2017

### CODE REVIEW

File: devtools/gotemplatelinter/translation_key_rule_special_cases.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	"dialog-close-btn",
+	"authflowv2/__error_account",
+	"page-navbar",
+	"__forgot_password_alternative",
+	"__settings_profile_item",
+	"__settings_profile_date_item",
+	"__settings_profile_address_item",
+	"__settings_profile_locale_item",
+	"__settings_gender_edit_custom_gender_input",
+}
```
Comment (louischan-oursky) [inline devtools/gotemplatelinter/translation_key_rule_special_cases.go:26]: Remove these.

File: devtools/gotemplatelinter/translation_key_rule_special_cases.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+var allowedVariableKeys []string = []string{
+	"$label",
+	"$labelKey",
+}
```
Comment (louischan-oursky) [inline devtools/gotemplatelinter/translation_key_rule_special_cases.go:31]: Remove these.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

File: devtools/gotemplatelinter/translation_key_rule_special_cases.go (hunk 1 above)
Comment (pkong-ds) [reply devtools/gotemplatelinter/translation_key_rule_special_cases.go]: Picked 2, please suggest if otherwise~

**Comment by pkong-ds** [comment]
Updated, thanks!

File: devtools/gotemplatelinter/translation_key_rule_special_cases.go (hunk 1 above)
Comment (louischan-oursky) [reply devtools/gotemplatelinter/translation_key_rule_special_cases.go]: I mean 3.

File: devtools/gotemplatelinter/translation_key_rule_special_cases.go (hunk 2 above)
Comment (louischan-oursky) [reply devtools/gotemplatelinter/translation_key_rule_special_cases.go:19]: I mean 3 here as well.

**Comment by pkong-ds** [comment]
Updated with 3, thanks!

**Pull request merged**